### ISSUE TITLE
Case Status?

### ISSUE THREAD

**Issue by kylebakerio** [bumbeishvili/covid19-daily-data#6]
On the worldometer site, it lists case status numbers (critical, mild, dead, recovered) as totals. Could you include this data as well?

**Comment by bumbeishvili** [comment]
@kylebakerio 

I think yes, but I can only include it in the `misc` folder

BTW, `recovered` and `deaths` already included, I can add `critical` as well, mild conditions will be calculatable

**Comment by kylebakerio** [comment]
Yeah, realized that afterwards. Just that Critical/Mild distinction that would be a nice-to-have.

**Comment by bumbeishvili** [comment]
Added critical number of patients  already - https://github.com/bumbeishvili/covid19-daily-data/blob/master/misc/april-12.csv

As I said I will add it only in a misc folder, since it won't increase my workload, if I figure out how to automate everything, then I will have it in the main dataset as well
